cli/template/addons/env/auth-prisma-schema.mjs

```diff
@@ -8,7 +8,10 @@ import { z } from "zod";
 export const serverSchema = z.object({
   DATABASE_URL: z.string().url(),
   NODE_ENV: z.enum(["development", "test", "production"]),
-  NEXTAUTH_SECRET: z.string().min(1),
+  NEXTAUTH_SECRET:
+    process.env.NODE_ENV === "production"
+      ? z.string().min(1)
+      : z.string().optional(),
```

Comment:
does `z.string().optional()` behave differently than just `z.string()` in this context?

Reply:
Should be .min(1).optional() I think.

We want it to be okay to omit the secret completely (which it is when it's commented out) but if it's defined, it can't be empty.

Reply:
> does `z.string().optional()` behave differently than just `z.string()` in this context?

Yes, optional means it can be commented out (I.e undefined)